erpnext/public/js/controllers/accounts.js

```
@@ -5,61 +5,69 @@
-			onload: function(frm) {
-				if(frm.get_field("taxes")) {
-					frm.set_query("account_head", "taxes", function(doc) {
-						if(frm.cscript.tax_table == "Sales Taxes and Charges") {
+			onload: function (frm) {
+				if (frm.get_field("taxes")) {
+					frm.set_query("account_head", "taxes", function (doc) {
+						if (frm.cscript.tax_table == "Sales Taxes and Charges") {
 							var account_type = ["Tax", "Chargeable", "Expense Account"];
 						} else {
-							var account_type = ["Tax", "Chargeable", "Income Account", "Expenses Included In Valuation"];
+							var account_type = [
```

Biome (2.1.2)

[error] Shouldn't redeclare 'account_type'. Consider to delete it or rename it.

'account_type' is defined here:


(lint/suspicious/noRedeclare)